srcpkgs/python3-text-unidecode/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-text-unidecode'
 pkgname=python3-text-unidecode
-version=1.2
-revision=4
+version=1.3
+revision=1
 wrksrc="text-unidecode-${version}"
 build_style=python3-module
-pycompile_module="text_unidecode"
 hostmakedepends="python3-setuptools"
-depends="python3"
```

Comment:
Did you remove this by mistake?

Reply:
I am not sure what you mean, $depends are often autodetected from the ELF header, but python packages are just scripts that obviously don't have ELF headers.

Python packages often include depends="python3-something" which depends on python3 transitively, so python3 doesn't need to be explicitly mentioned.

A few months ago I went over all python packages that were missing the python3 dependency and added it. You can check dependencies of the resulting package with xbps-query -x or a diff from the last version with xpkgdiff -x or in the CI (which runs xpkgdiff).